### PR TITLE
Feat/52/add build command

### LINKED ISSUE #52: Add build command to generate html page from markdown

### PR DESCRIPTION
**Pull request by Webblitchy**
Close #52

### CODE REVIEW

File: src/main/java/ch/heigvd/dil/utils/FileHandler.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    StringBuilder content = new StringBuilder();
+    try {
+      BufferedReader br = new BufferedReader(new FileReader(file, StandardCharsets.UTF_8));
+
+      String line;
+      while ((line = br.readLine()) != null) {
+        content.append(line).append("\n");
+      }
+    } catch (IOException e) {
+      return null;
```
Comment (Luhrel) [inline src/main/java/ch/heigvd/dil/utils/FileHandler.java:42]: Just throw it !

**Review by Luhrel — CHANGES_REQUESTED** [review]

File: src/main/java/ch/heigvd/dil/cli_cmds/BuildCmd.java [hunk 2]
```diff
@@ -1,13 +1,127 @@
...
+    // Crée le dossier build
+    File buildDir = new File(path, "build");
+    if (buildDir.exists()) {
+      status = FileHandler.delete(buildDir);
+      if (!status) {
+        System.err.println("Error: Could not delete the build folder.");
+        return 1;
+      }
+    }
+    buildDir.mkdir();
```
Comment (Luhrel) [inline src/main/java/ch/heigvd/dil/cli_cmds/BuildCmd.java:52]: What if it fails ?

File: src/main/java/ch/heigvd/dil/cli_cmds/BuildCmd.java [hunk 3]
```diff
@@ -1,13 +1,127 @@
...
+  /**
+   * Explore les fichiers Markdown du dossier courant et les convertit en HTML dans le dossier build
+   *
+   * @param folderPath le chemin du dossier courant
+   * @return true si tout s'est bien passé, false sinon
+   */
+  private boolean recursiveExploration(String folderPath) {
+    boolean status;
+
+    for (File f : new File(path, folderPath).listFiles()) {
```
Comment (Luhrel) [inline src/main/java/ch/heigvd/dil/cli_cmds/BuildCmd.java:75]: What if the `new FIle(...)` fails ?

**Review by Luhrel — CHANGES_REQUESTED** [review]

**Review by BasileCuneo — APPROVED** [review]
seems good

**Review by Luhrel — APPROVED** [review]

**Pull request merged**